charts/cluster-autoscaler/templates/configmap.yaml

```diff
@@ -0,0 +1,416 @@
+{{- if or (eq .Values.cloudProvider "kwok") }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.kwokConfigMapName | default "kwok-provider-config" }}
+  namespace: {{ .Release.Namespace }}
+data: 
+  config: |-
+    # if you see '\n' everywhere, remove all the trailing spaces
+    apiVersion: v1alpha1
+    readNodesFrom: configmap # possible values: [cluster,configmap]
+    nodegroups:
+      # to specify how to group nodes into a nodegroup
+      # e.g., you want to treat nodes with same instance type as a nodegroup
+      # node1: m5.xlarge
+      # node2: c5.xlarge
+      # node3: m5.xlarge
+      # nodegroup1: [node1,node3]
+      # nodegroup2: [node2]
+      fromNodeLabelKey: "kwok-nodegroup"
+      # you can either specify fromNodeLabelKey OR fromNodeAnnotation
+      # (both are not allowed)
+      # fromNodeAnnotation: "eks.amazonaws.com/nodegroup"
+    nodes:
+      # gpuConfig:
+      #   # to tell kwok provider what label should be considered as GPU label
+      #   gpuLabelKey: "k8s.amazonaws.com/accelerator"
+      #   availableGPUTypes:
+      #     "nvidia-tesla-k80": {}
+      #     "nvidia-tesla-p100": {}
+    configmap:
+      name: kwok-provider-templates
+    kwok: {} # default: fetch latest release of kwok from github and install it
+    # # you can also manually specify which kwok release you want to install
+    # # for example:
+    # kwok:
+    #   release: v0.3.0
+    # # you can also disable installing kwok in CA code (and install your own kwok release)
+    # kwok:
+    #   install: false (true if not specified)
+--- 
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kwok-provider-templates
```

Comment:
This is a sample `kwok-provider-templates` which helps users to get started with the kwok provider immediately (`kwok-provider-config` is in tune with this configmap).